pkg/detectors/easyinsight/easyinsight.go

```diff
@@ -24,8 +26,8 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"easyinsight", "easy-insight"}) + `\b([0-9Aa-zA-Z]{20})\b`)
-	idPat  = regexp.MustCompile(detectors.PrefixRegex([]string{"easyinsight", "easy-insight"}) + `\b([a-zA-Z0-9]{20})\b`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"easyinsight", "easy-insight", "key"}) + `\b([0-9a-zA-Z]{20})\b`)
+	idPat  = regexp.MustCompile(detectors.PrefixRegex([]string{"easyinsight", "easy-insight", "id"}) + `\b([a-zA-Z0-9]{20})\b`)
```

Comment:
@kashifkhan0771 'key' and 'id' are very generic keywords. Using them will increase the chances of false positive cases

Reply:
I kept them because the data chunk contains the keyword `easyinsight` or `easy-insight`, and this will allow us to search for a key or ID within that chunk. This approach should increase the likelihood of detecting the key and ID in the data which has the keywords for this detector.